### DISCUSSION TITLE
What comes next in this sequence? 📅

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1153] [Galaxy Trivia] [answered]
```
Y  Y  H  L  Y  E  Y  T ?
```

**Comment by CelioAmaral** [comment, marked as answer]
R, R, R, R
reason is they are initials of the ending letters of the months eg.
JanuarY FebreuarY MarcH ApriL MaY JunE JulY AugusT SeptembeR OctobeR NovembeR DecembeR
Y  Y  H  L  Y  E  Y  T R R R R

**Reply by ParanoidUser** [reply]
YaRrr! 😊
